SDL/Triplet.h

```diff
@@ -89,6 +95,9 @@ namespace SDL {
     Buf<TDev, FPX> betaIn_buf;
     Buf<TDev, FPX> betaOut_buf;
     Buf<TDev, FPX> pt_beta_buf;
+    Buf<TDev, FPX> Circle_Radius_buf;
+    Buf<TDev, FPX> Circle_CenterX_buf;
+    Buf<TDev, FPX> Circle_CenterY_buf;
```

Comment:
please keep lower case and camelCase as in the other members

Reply:
Addressed, thank you!